app/allocations/middleware/set-results.allocations.js

```diff
@@ -3,34 +3,24 @@ const allocationService = require('../../../common/services/allocation')
 
 async function setResultsAllocations(req, res, next) {
   const hasAssignerPermission = req.canAccess('allocation:person:assign')
-  const query = req.query
-
   const displayConfig = {
     showFromLocation: !hasAssignerPermission,
     showRemaining: hasAssignerPermission,
-    query,
+    query: req.query,
   }
 
   try {
-    const [activeAllocations, cancelledAllocations] = (
-      await Promise.all([
-        allocationService.getActive(req.body.allocations),
-        allocationService.getCancelled(req.body.allocations),
-      ])
-    ).map(response => response.flat())
+    const results = await allocationService.getByDateAndLocation(
+      req.body.allocations
+    )
 
     req.results = {
-      active: activeAllocations,
-      cancelled: cancelledAllocations,
+      active: results,
+      cancelled: [],
```

Comment:
Is there any reason for the cancelled property here any more? (Likewise for the Single Requests)

Reply:
So I believe this is still being used by the "download"...or maybe it's not actually?

But I think I might investigate and clean that up in a separate PR...

Reply:
Pretty sure we're not

https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/f826769d604ab6ff3137fde5a7ebceb9b93f13d6/common/services/move.js#L199

But yeah, separate PR is a good idea. I was just mentioning it…

Reply:
Yeah good spot. Lemme take a quick look. Might be able to do as a commit on this one as is kinda related...